.github/workflows/pull-request-management.yml

```diff
@@ -315,37 +316,60 @@ jobs:
           cd ansible_collections/arista/cvp/
           ansible-lint --force-color --strict -v
 
+  # ----------------------------------- #
+  # Build Collection
+  # ----------------------------------- #
+  build_collection:
+    name: Build Ansible collection
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-python@v5
+        with:
+          python-version: |
+            3.12
+      - uses: actions/checkout@v4
+      - name: "Install Python & Ansible requirements"
+        run: |
+          # distlib is required when using manifest
+          pip install "ansible-core<2.19.0" distlib --upgrade
+      - name: "Build Ansible collection"
+        run: |
+          ansible-galaxy collection build -vvv --force ansible_collections/arista/cvp
+      - name: Upload collection
+        uses: actions/upload-artifact@v4
+        with:
+          name: cvp-collection
+          path: ./arista-cvp-*.tar.gz
+
   # ----------------------------------- #
   # Galaxy Importer
   # ----------------------------------- #
   galaxy_importer:
     name: Test galaxy-importer
-    runs-on: ubuntu-20.04 # Older version to be compatible with old python
-    env:
-      PY_COLORS: 1 # allows molecule colors to be passed to GitHub Actions
-      ANSIBLE_FORCE_COLOR: 1 # allows ansible colors to be passed to GitHub Actions
-      GALAXY_IMPORTER_CONFIG: galaxy-importer/galaxy-importer.cfg
+    runs-on: ubuntu-latest
+    needs: [build_collection]
     steps:
       - uses: actions/setup-python@v5
         with:
           python-version: |
-            3.10
+            3.11
```

Comment:
Any reason to build with 3.12 but run galaxy importer with 3.11?

Reply:
new galaxy version support 3.12, so we are good! Updated in [d9f300e](https://github.com/aristanetworks/ansible-cvp/pull/726/commits/d9f300e3283f718c2727ae85cea0aecf794f339e)